site/helm/get-started.md

```diff
@@ -59,23 +59,34 @@ Add the Flux repository of Weaveworks:
 helm repo add weaveworks https://weaveworks.github.io/flux
 ```
 
-In this next step you install Weave Flux using `helm`. Simply:
+In this next step you install Weave Flux using `helm`. Simply
 
  1. Fork [flux-helm-test](https://github.com/weaveworks/flux-helm-test)
     on Github and
- 1. install Weave Flux and its Helm Operator by specifying your fork
+ 1. Install Weave Flux and its Helm Operator by specifying your fork
     URL:
 
-    *Just make sure you replace `YOURUSER` with your GitHub username
-    in the command below:*
-    ```sh
-    helm install --name flux \
-    --set helmOperator.create=true \
-    --set git.url=ssh://git@github.com/YOURUSER/flux-helm-test \
-    --set git.chartsPath=charts \
-    --namespace flux \
-    weaveworks/flux
-    ```
+      *Just make sure you replace `YOURUSER` with your GitHub username
+      in the command below:*
+      
+    - Using a public git repo from `bitbucket.com`, `github.com` or `gitlab.com`:
+    
+      ```sh
+      helm install --name flux \
+      --set helmOperator.create=true \
+      --set git.url=ssh://git@github.com/YOURUSER/flux-helm-test \
+      --set git.chartsPath=charts \
+      --namespace flux \
+      weaveworks/flux
+      ```
+      
+    - Using a private git repo:
+       
+      When deploying from a private repo, the known_hosts of the git repo needs 
+      to be configured into a kubernetes secret so that `StrictHostKeyChecking` is respected.
```

Comment:
It's a ConfigMap in the chart -- nitpick!

Reply:
yea, I'm writing the docs now for the README.md and I noticed that -- those are not exactly secret!